tests/integration/query/simple/with_multiple_types_test.go

```diff
@@ -0,0 +1,323 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package simple
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestSimple_WithSevenDummyTypesBefore(t *testing.T) {
```

Comment:
nitpick: would be nice to have "expectation" part in the test name:
`TestSimple_WithSevenDummyTypesBefore_ShouldSucceed` or something like this.